java/src/main/java/com/xxAMIDOxx/xxSTACKSxx/menu/api/v1/impl/QueryMenuControllerImpl.java

```diff
@@ -33,7 +33,8 @@
 
     private MenuQueryService menuQueryService;
 
-    public QueryMenuControllerImpl(DomainToDtoMapper mapper, MenuQueryService menuQueryService) {
```

Comment:
Can the `MenuQueryService` be injected?

Reply:
the common practice now is using constructor injection.